src/components/badge/styles/SBadgeIcon.tsx

```diff
@@ -4,43 +4,33 @@ import { BadgePosition } from "../BadgePosition";
 
 interface ISBadgeIcon {
   position: BadgePosition;
-  color?: string;
-  backgroundColor?: string;
   disabled?: boolean;
+  height: number;
 }
 
 const Bronze = css<ISBadgeIcon>`
   position: absolute;
+
   z-index: 10;
-  top: 0;
-  left: ${(props) => (props.position == BadgePosition.TopLeft ? "0" : "")};
-  right: ${(props) => (props.position == BadgePosition.TopRight ? "0" : "")};
-  transform: translateX(${(props) => (props.position == BadgePosition.TopLeft ? "-35%" : "35%")}) translateY(-35%);
+  top: ${(props) => `-${props.height / 2}px`};
+  ${(props) => (props.position == BadgePosition.TopLeft ? `left: -${props.height / 2}px` : "")};
+  ${(props) => (props.position == BadgePosition.TopRight ? `right: -${props.height / 2}px` : "")};
 
-  color: ${(props) => (props.color ? props.color : "white")};
-  font-size: 12px;
+  color: var(--ui-01);
+  font-size: 9px;
+  font-weight: 700;
```

Comment:
@nicolasshiken  instead of hardcoding the font-weight and font-size i think you should find the fond already defined in the constants somewhere in the generated code from the design tokens 

you can use that here like this : `${CSSSomeFontSnippet}`